app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -20,11 +20,12 @@ function require_ev_one() {
     exit 1
 }
 
-require_ev_one ETCD_IMG ETCD_TAG
+require_ev_one ETCD_IMG ETCD_IMG_TAG
```

Comment:
I kept this from rolling to Stable so that we could break it. I'm fine with the change.